state_manager/tests/test_messages.py

```diff
@@ -423,11 +428,19 @@ def test_valid_handle_get_sync_pin_state_message(ot3_state: OT3State) -> None:
     HIGH_MESSAGE = SyncPinMessage(SyncPinState.HIGH)
     LOW_MESSAGE = SyncPinMessage(SyncPinState.LOW)
     ack = handle_message(HIGH_MESSAGE.to_bytes(), ot3_state)
-    assert ack == HIGH_MESSAGE.to_bytes()
+    assert ack is not None
+    assert ack.broadcast
     assert ot3_state.get_sync_pin_state()
-    assert handle_message(GetSyncPinStateMessage().to_bytes(), ot3_state) == b"1"
+    ack = handle_message(GetSyncPinStateMessage().to_bytes(), ot3_state)
+    assert ack is not None
+    assert not ack.broadcast
+    assert ack.to_bytes() == b"\x031"
```

Comment:
Can you set b"\x031" to a constant? I am wanting context of what b"\x031" represents.